node/task.ts

```diff
@@ -897,6 +897,10 @@ export function find(findPath: string, options?: FindOptions): string[] {
                 }
             }
             else {
+                if (!fs.existsSync(item.path)) {
+                    debug(`File "${item.path}" has not been found and will be skipped.`);
```

Comment:
Could it make sense to change it to something like 'File "${item.path}" seems to be removed during find operation execution - so skipping it.'?
Currently it is unclear why do we mention some not existing path in logs.